src/api/app/models/history_element.rb

```diff
@@ -19,14 +19,18 @@ def color
 
 class HistoryElement::Request < ::HistoryElement::Base
   belongs_to :review,
-             -> { where(history_elements: { type: 'HistoryElement::RequestReviewAdded' })
-                    .joins('JOIN history_elements ON history_elements.description_extension = reviews.id') },
              class_name: '::Review',
-             foreign_key: :description_extension
+             foreign_key: :description_extension,
+             scope: :restrict_to_review_added
 
   self.description = 'Request was updated'
   self.abstract_class = true
 
+  def self.restrict_to_review_added
```

Comment:
Sounds like a [scope](https://guides.rubyonrails.org/active_record_querying.html#scopes)

Reply:
Oh sorry, I abandoned the idea of extracting that lambda.